frontend/src/components/SubmitFormDialog.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button, Dialog, Typography } from '@equinor/eds-core-react'
+import styled from 'styled-components'
+
+const Wrapper = styled.div`
+    display: grid;
+    column-gap: 16px;
+    grid-template-columns: repeat(2, fit-content(100%));
+    justify-content: end;
+    justify-self: end;
+`
+
+interface SubmitFormDialogProps {
+    ritm: string
+    onConfirmClick: () => void
+}
+
+export const SubmitFormDialog = ({ ritm, onConfirmClick }: SubmitFormDialogProps) => {
+    return (
+        <Dialog>
+            <Dialog.Title>Order Successful!</Dialog.Title>
```

Comment:
Successful -> successful?

Reply:
Major first letter in each word is common in titles, isn't it?

Reply:
https://eds.equinor.com/guidelines/typography/
says that it should be sentence-case as you suggested, will change.